frontend/core/blocks/image-data-extraction/specs.json

```diff
@@ -0,0 +1,58 @@
+{
+  "information": {
+    "id": "data-extraction",
+    "name": "Data Extraction",
+    "description": "Block to extract data from images",
```

Comment:
Please add a more comprehensive description, for example, like the description that is in the function itself.